src/Tribe/Editor.php

```diff
@@ -38,7 +38,14 @@ public function hook() {
 		add_action( 'admin_init', [ $this, 'assets' ] );
 
 		// Add Block Categories to Editor
-		add_action( 'block_categories', [ $this, 'block_categories' ], 10, 2 );
+		global $wp_version;
+		if ( ! class_exists( 'WP_Block_Editor_Context' ) || version_compare( $wp_version, '5.8', '<' ) ) {
```

Comment:
The `version_compare` check is cheaper and we're using the `class_exists`, here, just to run the same check on the WordPress version. Drop the `class_exist` and just run the `version_compare`?

Reply:
Yeah, the class_exists check is the "official" recommended one - but I found it unreliable. I can drop it.